dotnet/src/Agents/OpenAI/FileReferenceContent.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel.Agents.OpenAI;
+
+/// <summary>
+/// Content type to support file references.
+/// </summary>
+public class FileReferenceContent : KernelContent
```

Comment:
Should this be in SK.Abstractions?

Reply:
Same comment as for the `AnnotationContent` class + @RogerBarreto

Reply:
Makes sense, will move.